static/.gitkeep

```diff
@@ -0,0 +1 @@
+Placeholder file to ensure that the static/ folder is kept by Git.
```

Comment:
You don't need to put the root static folder under version control. If the folder doesn't exist it will be created the first time collectstatic is run.